R/glue.R

```diff
@@ -287,12 +287,21 @@ trim <- function(x) {
 
 #' @export
 print.glue <- function(x, ..., sep = "\n") {
-  x[is.na(x)] <- style_na(x[is.na(x)])
-
-  if (length(x) > 0) {
-    cat(x, ..., sep = sep)
+  if (length(x) == 0) {
+    cat("<Length-0 glue vector>\n")
```

Comment:
<img width="266" alt="Screen Shot 2023-01-27 at 4 03 14 PM" src="https://user-images.githubusercontent.com/19150088/215198187-31c19224-8595-424a-87cc-eda71be931ed.png">

You could use the same idea as vctrs and always show `<glue[n]>` so the 0 length case feels more natural